simple/values.yaml

```diff
@@ -114,6 +114,9 @@ backendConfig:
 nginx:
   image: 'you need to pass in a value for nginx.image to your helm chart'
 
+  # Magic header for CDN (string). If not empty then it goes to varnish and nginx config
```

Comment:
Also add this "If this value is set services (varnish and nginx) will check request header and return not-allowed response unless request will have s-proxy-auth header with the value defined here"

Reply:
Decription is too long for people who won't ever use it, my proposal is this:
```yaml
# Requires "X-Proxy-Auth" header from upstream when defined. 
```